rpc_jobs/standard_job_fi.yml

```diff
@@ -0,0 +1,204 @@
+- job:
+    name: 'Component-FI-Test-Trigger'
+    project-type: pipeline
+    concurrent: true
+    triggers:
+      - github-pull-request:
+          org-list:
+            - rcbops
+          github-hooks: true
+          trigger-phrase: '.*recheck_(cit_)?all.*|.*recheck_cit_fi_test'
+          only-trigger-phrase: false
+          auth-id: "github_account_rpc_jenkins_svc"
+          status-context: 'CIT/re_fi_test'
+          cancel-builds-on-update: true
+    properties:
+      - github:
+          url: "https://github.com/mattt416/rpc-metadata"
+      - build-discarder:
+          days-to-keep: 30
+    parameters:
+      - rpc_gating_params
+    dsl: |
+      library "rpc-gating@${RPC_GATING_BRANCH}"
+      common.globalWraps(){
+        dir("${env.WORKSPACE}/rpc-metadata") {
+          common.clone_with_pr_refs()
+
+          sh """#!/bin/bash -xe
+              virtualenv --python python3 .venv3
+              set +x; . .venv3/bin/activate; set -x
+              pip install -c constraints.txt -r requirements.txt
+              pushd rpc_component
+                  pip install .
+              popd
+          """
+          String component_text = sh (script: """#!/bin/bash -xe
+              set +x; . .venv3/bin/activate; set -x
+              from=\$(git log --pretty=format:%H  --no-merges -n 1 --skip 1)
+              to=\$(git log --pretty=format:%H --no-merges -1)
+              component --releases-dir . compare --from \${from} --to \${to}
+          """,
+          returnStdout: true)
+      
+          def component_yaml = readYaml text: component_text
+
```

Comment:
Hi @git-harry , currently the component tool returns a structure that looks like this:

```
rpc-product-1:
  added:
    releases:
    - versions:
      - sha: f9a21de1f50ca708bab7e84eeaa055aa32a72063
        version: r1.3.1
  deleted: {}
```

Do you think it'd be better to have it return something like this instead?

```
name: rpc-product-1:
added:
  releases:
  - series: master
    versions:
    - sha: f9a21de1f50ca708bab7e84eeaa055aa32a72063
      version: r1.3.1
deleted: {}
```